WIP-CHANGELOG.md

```diff
@@ -27,6 +28,7 @@ Grafana 8.0.1 and Prometheus 2.27.1.
 - ck8s providers command
 - ck8s flavors command
 - Added script to make it easier to generate secrets
+- openid migration script
```

Comment:
Remove this line, it is enough if it's mentioned in the upgrade docs.
```suggestion
```

Reply:
Done!